ui/src/app/views/notebook-list/component.ts

```diff
@@ -81,6 +92,21 @@ export class NotebookListComponent implements OnInit, OnDestroy {
           this.cohortsError = true;
         });
     this.loadNotebookList();
+    this.profileService.getPageVisits().subscribe(
```

Comment:
This is already on the profile right? Can we just read it from there to avoid the extra call?